core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -2742,6 +2742,11 @@ private[spark] object Utils extends Logging {
     }
   }
 
+  def getTimeZone: TimeZone = {
```

Comment:
Could you please image that this method will be used outside scope of the current issue?
What's about users who does not have spark history at all?

The name of method, the class and the package does not tell us that this method somehow related ONLY to history server, such API makes me easy to do wrong assumptions.